### ISSUE TITLE
Creating the same training set as used in the paper.

### ISSUE THREAD

**Issue by saksham15082** [PeterWang512/CNNDetection#1]
Is it possible to get a method to curate the exact same training set as used in the paper?

**Comment by PeterWang512** [comment]
I haven't check whether my training set generation code behave deterministically (e.g., generating same images via different machines, or extracting the same lsun images from lmdb). Meanwhile, we're looking at potential releases of the training code and training set.

However, if you would like to train your own progan model, you can generate progan images directly from their [repo](https://github.com/tkarras/progressive_growing_of_gans). Moreover, you can also make lsun real images by downloading the lmdb files and process it using the `create_lsun` function from `dataset_tool.py` in the progan repo. This is exactly how we collected our training set and test set. However, if you make your own training set, you will need to create your own progan/stylegan/stylegan2 testset to avoid train/test leakage in the real images. Thanks!
